### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+<head>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
+</head>
+<body>
+  <div class="content">
+  <header class="site-header">
```
Comment (takrami) [inline index.html]: Please fix indentation

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    <div class="site-wrapper">header</div>
+  </header>
+  <main class="site-main">
+    <section class="first-section">
```
Comment (takrami) [inline index.html]: Each section must have heading. Please see this links: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

File: index.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      <div class="site-wrapper">
+        <p>text</p>
+      </div>
+    </section>
+    <section class="second-section">
```
Comment (takrami) [inline index.html]: Missing heading.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+      <div class="site-wrapper">
+        <header class="second-section-title">header</header>
+        <p>text</p>
+      </div>
+    </section>
+    <section class="third-section">
```
Comment (takrami) [inline index.html]: Missing heading.

**Review by takrami — CHANGES_REQUESTED** [review]

File: index.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+              <header class="title-feature1">header</header>
+              <p class="content-feature1">text</p>
+              <ul class="link-list">
+                <li class="link-list-item">1</li>
+                <li class="link-list-item">2</li>
+                <li class="link-list-item">3</li>
+              </ul>
+          </li>
+          <li class="project-list-item">item 2
+            <img>
```
Comment (blacksrc) [inline index.html]: There are couple of empty `` tags here. Could you explain the reason or just remove it?

File: style.css [hunk 6]
```diff
@@ -0,0 +1,190 @@
+html, body {
+  height: 100%;
+}
+body {
```
Comment (blacksrc) [inline style.css]: blank line maybe?

File: style.css [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+  }
+
+  .project-list-item {
+    flex: 50%;
+  } 
+}
+
+/* Desktop */
+@media(min-width: 901px) {
+
```
Comment (blacksrc) [inline style.css]: remove this blank line?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+      <header class="seventh-section-header">header</header>
+      <ul class="for-more">
+        <li class="for-more-item">1</li>
+        <li class="for-more-item">2</li>
+        <li class="for-more-item">3</li> 
+      </ul>
+    </section>
+  </div>
+</div>
+  </main>
```
Comment (blacksrc) [inline index.html]: Something is off with the indentation here.

File: style.css [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+  border: 4px solid #000000;
+}
+
+.sixth-section-header {
+  text-align: center;
+  border: 2px solid red;
+}
+
+.skills {
+ border: 2px solid #000000;
```
Comment (blacksrc) [inline style.css]: 2 spaces.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,156 @@
+
```
Comment (blacksrc) [inline index.html]: why blank line here?

File: index.html [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
```
Comment (blacksrc) [inline index.html]: Is it possible to use `./style.css` instead? :thinking:

File: style.css [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+.for-more {
+  display: flex;
+  justify-content: space-around;
+}
+
+.for-more-item {
+  border: 1px solid #000000;
+}
+
+/* Tablets */
```
Comment (blacksrc) [inline style.css:173]: Nice that you write comments :+1:

File: style.css [hunk 13]
```diff
@@ -0,0 +1,190 @@
...
+.content {
+  flex: 1 0 auto;
+}
+
+.site-header {
+  background-color: #f2b4f4;
+}
+
+.site-main {
+background-color: #F4F4F4;
```
Comment (blacksrc) [inline style.css]: indentation :see_no_evil:

**Comment by blacksrc** [comment]
Are these paddings intentional? `.skills`
image
Also you have it here `.for-more` and `.app-list`

File: style.css (hunk 6 above)
Comment (blacksrc) [reply style.css]: Please remove `body`'s margin.

File: style.css [hunk 14]
```diff
@@ -0,0 +1,190 @@
...
+}
+
+.site-footer {
+  background-color: #072BCE;
+  color: #ffffff;
```
Comment (blacksrc) [inline style.css]: Either go for the upper case letters or lower case letters.
so:
```css
#072bce
#ffffff
```
or
```css
#072BCE
#FFFFFF
```

File: index.html (hunk 1 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 5 above)
Comment (hedita) [reply index.html]: I removed it.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,153 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
```
Comment (blacksrc) [inline index.html]: should be `Portfolio` with the capital `P`.

File: style.css [hunk 16]
```diff
@@ -0,0 +1,192 @@
...
+  text-align: center;
+  border: 2px solid red;
+}
+
+.skills {
+  border: 2px solid #000000;
+}
+
+.skills-item {
+  border: 1px solid red;
```
Comment (blacksrc) [inline style.css:152]: ?

File: index.html (hunk 4 above)
Comment (hedita) [reply index.html]: Done.

File: style.css (hunk 6 above)
Comment (hedita) [reply style.css]: Done.

File: style.css (hunk 7 above)
Comment (hedita) [reply style.css]: Done.

File: style.css (hunk 9 above)
Comment (hedita) [reply style.css]: Done.

File: index.html (hunk 10 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 11 above)
Comment (hedita) [reply index.html]: I didn't get it.

File: style.css (hunk 12 above)
Comment (hedita) [reply style.css:173]: Thanks.

File: style.css (hunk 13 above)
Comment (hedita) [reply style.css]: Done.

File: style.css (hunk 14 above)
Comment (hedita) [reply style.css]: Done.

File: index.html (hunk 15 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 11 above)
Comment (blacksrc) [reply index.html]: Never mind, it doesn't matter here.

File: index.html [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+        <li class="for-more-item">2</li>
+        <li class="for-more-item">3</li> 
+      </ul>
+    </section>
+  </div>
+    </main>
+  <footer class="site-footer">
+    <div class="site-wrapper">footer</div>
+  </footer>
+  </div>
```
Comment (blacksrc) [inline index.html]: indentation.

File: index.html [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+        </ul>
+      </div> 
+    </section>
+    <div class="site-wrapper">
+      <section class="fourth-section">
+        <h2 class="title-toughts">header</h2>
+        <p class="about-thoughts">text</p>
+      </section>
+    </div>
+  <div>
```
Comment (blacksrc) [inline index.html]: indentation.

File: style.css [hunk 19]
```diff
@@ -0,0 +1,192 @@
...
+.project-list {
+  padding: 0;
+  gap: 25px;
+}
+
+.project-list-item {
+  border: 2px solid #000000;
+}
+
+.feature1 {
```
Comment (blacksrc) [inline style.css]: why `.feature1`? why not just `.feature`?

File: style.css (hunk 7 above)
Comment (blacksrc) [reply style.css]: This blank line is still there.

File: index.html (hunk 11 above)
Comment (hedita) [reply index.html]: Ok.

File: style.css (hunk 19 above)
Comment (hedita) [reply style.css]: Fixed.

File: index.html (hunk 18 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 17 above)
Comment (hedita) [reply index.html]: Done.

File: style.css (hunk 7 above)
Comment (hedita) [reply style.css]: You mean line 184?

File: index.html (hunk 8 above)
Comment (hedita) [reply index.html]: Done.

File: index.html [hunk 20]
```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (takrami) [inline index.html]: Please fix the indentation

File: index.html [hunk 21]
```diff
@@ -0,0 +1,157 @@
...
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+</head>
+<body>
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 22]
```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (takrami) [inline index.html]: These tags need to be close. They are self-closing tag.

File: index.html [hunk 23]
```diff
@@ -0,0 +1,157 @@
...
+        <h2>Heading</h2>
+        <div class="site-wrapper">
+          <p>text</p>
+        </div>
+      </section>
+      <section class="second-section">
+        <h2>Heading</h2>
+        <div class="site-wrapper">
+          <header class="second-section-title">header</header>
+            <p>text</p>
```
Comment (takrami) [inline index.html:19]: Indentation.

File: index.html [hunk 24]
```diff
@@ -0,0 +1,157 @@
...
+          <ul class="project-list">
+            <li class="project-list-item">item 1
+              <section class="feature">
+                <h2 class="title-feature">header</h2>
+                  <p class="content-feature">text</p>
+                  <ul class="link-list">
+                    <li class="link-list-item">1</li>
+                    <li class="link-list-item">2</li>
+                    <li class="link-list-item">3</li>
+                  </ul>
```
Comment (takrami) [inline index.html:42]: Indentation.

File: index.html [hunk 25]
```diff
@@ -0,0 +1,157 @@
...
+              <h2 class="title-feature">header</h2>
+              <p class="content-feature">text</p>
+              <ul class="link-list">
+                <li class="link-list-item">1</li>
+                <li class="link-list-item">2</li>
+                <li class="link-list-item">3</li>
+                <li class="link-list-item">4</li>
+              </ul>
+            </section>
+          </li>
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 26]
```diff
@@ -0,0 +1,157 @@
...
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 27]
```diff
@@ -0,0 +1,157 @@
...
+          <li class="app-list-item">2</li>
+          <li class="app-list-item">3</li>
+        </ul>
+        <p class="app-feature">text</p>
+        <ul class="app-list">
+          <li class="app-list-item">1</li>
+          <li class="app-list-item">2</li>
+          <li class="app-list-item">3</li>
+        </ul>
+      </div>
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 28]
```diff
@@ -0,0 +1,157 @@
...
+          <li class="app-list-item">2</li>
+          <li class="app-list-item">3</li>
+        </ul>
+      </div> 
+      </section>
+      <div class="site-wrapper">
+      <section class="fourth-section">
+        <h2 class="title-toughts">header</h2>
+        <p class="about-thoughts">text</p>
+      </section>
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 29]
```diff
@@ -0,0 +1,157 @@
...
+    <div>
+      <div class="site-wrapper">
+        <section class="fifth-section">
+          <h2 class="fifth-section-header">header</h2>
+          <p class="fifth-section-text">text</p>
+          <p class="fifth-section-text">text</p>
+          <p class="fifth-section-text">text</p>
+        </section>
+      </div>
+    </div>
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 30]
```diff
@@ -0,0 +1,157 @@
...
+          <li class="for-more-item">1</li>
+          <li class="for-more-item">2</li>
+          <li class="for-more-item">3</li> 
+        </ul>
+      </section>
+    </div>
+    </main>
+  <footer class="site-footer">
+  <div class="site-wrapper">footer</div>
+  </footer>
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 31]
```diff
@@ -0,0 +1,157 @@
...
+  </div>
+</body>
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 32]
```diff
@@ -0,0 +1,157 @@
...
+        <ul class="for-more">
+          <li class="for-more-item">1</li>
+          <li class="for-more-item">2</li>
+          <li class="for-more-item">3</li> 
+        </ul>
+      </section>
+    </div>
+    </main>
+  <footer class="site-footer">
+  <div class="site-wrapper">footer</div>
```
Comment (takrami) [inline index.html]: Indentation.

File: index.html [hunk 33]
```diff
@@ -0,0 +1,157 @@
...
+            <li class="skills-item">4</li>
+            <li class="skills-item">5</li>
+            <li class="skills-item">6</li>
+          </ul>
+          <ul class="skills">
+            <li class="skills-item">1</li>
+            <li class="skills-item">2</li>
+          </ul>
+        </section>
+      </div>
```
Comment (takrami) [inline index.html]: Indentation

File: index.html [hunk 34]
```diff
@@ -0,0 +1,157 @@
...
+    <div class="site-wrapper">
+      <section class="seventh-section">
+        <h2 class="seventh-section-header">header</h2>
+        <ul class="for-more">
+          <li class="for-more-item">1</li>
+          <li class="for-more-item">2</li>
+          <li class="for-more-item">3</li> 
+        </ul>
+      </section>
+    </div>
```
Comment (takrami) [inline index.html]: Indentation.

**Review by takrami — CHANGES_REQUESTED** [review]

File: index.html (hunk 20 above)
Comment (hedita) [reply index.html]: fixed.

File: index.html (hunk 21 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 23 above)
Comment (hedita) [reply index.html:19]: Done.

File: index.html (hunk 22 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 24 above)
Comment (hedita) [reply index.html:42]: Done.

File: index.html (hunk 30 above)
Comment (hedita) [reply index.html]: Fixed.

File: index.html (hunk 31 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 32 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 33 above)
Comment (hedita) [reply index.html]: Fixed.

File: index.html (hunk 34 above)
Comment (hedita) [reply index.html]: I think it's right. Is it possible to check it again?

File: index.html [hunk 35]
```diff
@@ -0,0 +1,153 @@
...
+                  <h2 class="title-feature">header</h2>
+                  <p class="content-feature">text</p>
+                  <ul class="link-list">
+                    <li class="link-list-item">1</li>
+                    <li class="link-list-item">2</li>
+                    <li class="link-list-item">3</li>
+                  </ul>
+                </section>
+              </li>
+            <li class="project-list-item">item 2
```
Comment (takrami) [inline index.html]: indentation

File: index.html [hunk 36]
```diff
@@ -0,0 +1,153 @@
...
+            <li class="app-list-item">3</li>
+          </ul>
+        </div> 
+        </section>
+        <div class="site-wrapper">
+        <section class="fourth-section">
+          <h2 class="title-toughts">header</h2>
+          <p class="about-thoughts">text</p>
+        </section>
+        </div>
```
Comment (takrami) [inline index.html]: indentation

File: index.html [hunk 37]
```diff
@@ -0,0 +1,153 @@
...
+            <li class="app-list-item">3</li>
+          </ul>
+          <p class="app-feature">text</p>
+          <ul class="app-list">
+            <li class="app-list-item">1</li>
+            <li class="app-list-item">2</li>
+            <li class="app-list-item">3</li>
+          </ul>
+        </div> 
+        </section>
```
Comment (takrami) [inline index.html]: You need to check this part of the code more carefully to fix the indentation.

**Review by takrami — CHANGES_REQUESTED** [review]

File: index.html (hunk 35 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 36 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 37 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 36 above)
Comment (takrami) [reply index.html]: I can still see it.

Comment (hedita) [reply index.html]: Oh!
Sorry,I fixed it.

File: index.html [hunk 38]
```diff
@@ -0,0 +1,152 @@
...
+              <li class="app-list-item">3</li>
+            </ul>
+            <p class="app-feature">text</p>
+            <ul class="app-list">
+              <li class="app-list-item">1</li>
+              <li class="app-list-item">2</li>
+              <li class="app-list-item">3</li>
+            </ul>
+          </div> 
+          </section>
```
Comment (takrami) [inline index.html:101]: It needs to shift two spaces back. Also the whole code from line 97 till 145.

**Review by takrami — CHANGES_REQUESTED** [review]

File: index.html (hunk 38 above)
Comment (hedita) [reply index.html:101]: Done.

**Comment by takrami** [comment]
Nice job! The code is really clean and structured :100: :+1: :star:

**Review by takrami — APPROVED** [review]
Thanks for your hardworking :1st_place_medal:

**Review by blacksrc — APPROVED** [review]
:rocket: great stuff.

**Pull request merged**